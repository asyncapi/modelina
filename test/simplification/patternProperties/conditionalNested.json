{
    "patternProperties": { 
      "testPattern1": { 
        "type": "string"
      }
    },
    "if": {
        "patternProperties": { 
          "testPattern1": { 
            "const": "string"
          } 
        } 
    },
    "then": {
        "patternProperties": {
          "testPattern2": { 
            "type": "string"
          } 
        },
        "if": {
            "patternProperties": { 
              "testPattern2": { 
                "const": "string"
              } 
            } 
        },
        "then": {
            "patternProperties": { 
              "testPattern3": { 
                "type": "string"
              } 
            } 
        },
        "else": {
            "patternProperties": { 
              "testPattern4": { 
                "type": "string"
              } 
            } 
        }
    },
    "else": {
        "patternProperties": { 
          "testPattern5": { 
            "type": "string"
          } 
        } 
    }
}