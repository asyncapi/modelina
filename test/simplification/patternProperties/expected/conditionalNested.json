{
  "testPattern1": {  },
  "testPattern2": {  },
  "testPattern3": {  },
  "testPattern4": {  },
  "testPattern5": {  }
}